lib/win32/certstore/store_base.rb

```diff
@@ -101,9 +106,9 @@ def lookup_error(failed_operation = nil)
       # A certificate can be converted with `openssl x509 -in example.crt -out example.der -outform DER`
       def read_certificate_content(cert_path)
         unless (File.extname(cert_path) == ".der")
-          temp_file = shell_out("powershell.exe -Command $env:temp").stdout.strip.concat("\\TempCert.der")
-          shell_out("powershell.exe -Command openssl x509 -in #{cert_path} -outform DER -out #{temp_file}")
-          cert_path = temp_file
+          temp_file_path = shell_out_command("powershell.exe -Command $env:temp").stdout.strip.concat("\\TempCert.der")
+          shell_out_command("powershell.exe -Command openssl x509 -in #{cert_path} -outform DER -out $env:temp\\TempCert.der")
```

Comment:
Yes ... It would be there this line of code was already written and I just changed shell_out to shell_out_command and temp_file variable to temp_file_path to understand that this is returning file path. You are right directly $env:temp\TempCert.der can be used instead of executing the powershell command but then thought that this code is already been reviewed and merged so not to change it other than my changes.

Reply:
Yes I just missed that to change to temp_file_path will do this change.